hydromt_wflow/workflows/soilgrids.py

```diff
@@ -165,6 +167,71 @@ def pore_size_distrution_index_layers(ds, thetas):
     return ds_out
 
 
+def brooks_corey_layers(c_sl, soil_fn, wflow_layers, soildepth_cm):
+    wflow_thickness = wflow_layers.copy()
+    # Go from wflow layer thickness to soil depths (cumulative)
+    wflow_depths = np.cumsum(wflow_thickness)
+    # Check if the last wflow depth is less than 2000 mm (soilgrids limit)
+    if wflow_depths[-1] > 2000:
+        raise ValueError("The total depth of the wflow soil layers should be 2000 mm.")
```

Comment:
Add a remark here that this is a limitation of SoilGrids, not wflow.

```suggestion
        raise ValueError(
            "The total depth of the wflow soil layers should be 2000 mm, as the \
soilgrids data does not go deeper than 2 m."
        )
```